### ISSUE TITLE
Using ncurses 6 (libtinfo.so.6 instead of libtinfo.so.5)

### ISSUE THREAD

**Issue by SiebenCorgie** [GPUOpen-Tools/radeon_gpu_profiler#30]
Hi,

first of all I'm on archlinux and not ubuntu, I know that it's not supported, so a short "no" will do if there is no way.

Now the question: It seems that the Profiler is compiled agains ncurses 5.0, but the arch repositories only have ncurses 6.0. I tried to symlink libtinfo.so.5 to libtinfo.so but then I get the following runtime error:
```
./RadeonGPUProfiler: error while loading shared libraries: /usr/lib/libtinfo.so.5: file too short
```

which is fair I guess since they are two different version.

Is there any way for me to make this work or should I just wait for another release / roll back my ncurses package (if possible)?

**Comment by chesik-amd** [comment]
Hi Sieben,

We had a similar issue when running on Ubuntu 20.04. This issue has been fixed in RGP v1.8 which was released yesterday.  Can you try out the new release and let us know if you still see this issue on archlinux?

**Comment by SiebenCorgie** [comment]
Hi it works with the normal `ncurses` package now on my Archlinux.
Thanks!
